scipy/stats/mstats_basic.py

```diff
@@ -549,6 +549,42 @@ def _spearmanr_2cols(x):
         return SpearmanrResult(rs, prob)
 
 
+def _kendall_p_exact(n, c):
+    # Exact p-value, see Maurice G. Kendall, "Rank Correlation Methods" (4th Edition), Charles Griffin & Co., 1970.
+    if n <= 0:
+        raise ValueError
+    elif c < 0 or 4*c > n*(n-1):
+        raise ValueError
```

Comment:
This line probably should be something like:
  ValueError(f'c ({c}) must satisfy 0 <= 4*c <= n*(n-1) = {n*(n-1)}.')

Reply:
Thanks. Fixed!